### PR TITLE
Create Hacktoberfest Event Organizer Guide for Mattermost

### PR DESCRIPTION
**Pull request by jasonblais**
This is part of Mattermost-DigitalOcean-MLH partnership during Hacktoberfest, where each virtual Hacktoberfest event will have a dedicated Mattermost channel for them in https://hacktoberfest.cloud.mattermost.com/

### CODE REVIEW

File: contributors/contributors/hacktoberfest-mattermost-organizer-guide.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Hacktoberfest Event Organizer Guide for Mattermost
+
+This page gives you tips for running your Hacktoberfest event on the Mattermost platform.
+
+## Join your event channel
+
+Each [Hacktoberfest event](https://hacktoberfest.digitalocean.com/events) has a dedicated Mattermost channel for them in [https://hacktobefest.cloud.mattermost.com/](https://hacktobefest.cloud.mattermost.com/). To join your event channel:
```
Comment (justinegeffen) [inline contributors/contributors/hacktoberfest-mattermost-organizer-guide.md]:
```suggestion
Each [Hacktoberfest event](https://hacktoberfest.digitalocean.com/events) has a dedicated Mattermost channel on the [https://hacktobefest.cloud.mattermost.com/](https://hacktobefest.cloud.mattermost.com/) server. To join your event channel:
```

File: contributors/contributors/hacktoberfest-mattermost-organizer-guide.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+1. [Create your Mattermost account](https://hacktoberfest.cloud.mattermost.com/signup_email).
+2. Verify your email address.
+3. On the left-hand side, click the “Jump to…” bar and type your event name.
```
Comment (justinegeffen) [inline contributors/contributors/hacktoberfest-mattermost-organizer-guide.md]:
```suggestion
3. On the left-hand side, click the **Jump to…** bar and type your event name.
```

File: contributors/contributors/hacktoberfest-mattermost-organizer-guide.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+4. Select the resulting channel for your event.
+
+Once you’ve joined the channel, you’ll find the event time and location in the channel header, along with the event schedule posted by the **Hacktoberfest** bot:
+
+![](../.gitbook/assets/hacktoberfest-organizer-bot-message.png)
+
+## Connect with your community
+
+Mattermost is a great way to connect with your community before, during and after the event. It is also a great way to create an open, inclusive, and welcoming atmosphere to everyone.
```
Comment (justinegeffen) [inline contributors/contributors/hacktoberfest-mattermost-organizer-guide.md]:
```suggestion
Mattermost is a great way to connect with your community before, during, and after the event. It's also a great way to create an open, inclusive, and welcoming atmosphere for everyone.
```

File: contributors/contributors/hacktoberfest-mattermost-organizer-guide.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+Use the channel to facilitate conversations related to your event. For instance, what questions do people have? What open source contributions are they planning to make? What ideas do they have for Hacktoberfest?
+
+After the event, thank your attendees for taking the time to attend your Hacktoberfest meetup, and keep the conversation going throughout October.
+
+## Quick tips
+
+- Complete your profile in **Main Menu > Account Settings**, including your profile picture and name, so your community can make a more personal connection with you.
+- Join **Off-Topic** channel to have conversations with all Hacktoberfest event organizers and attendees.
```
Comment (justinegeffen) [inline contributors/contributors/hacktoberfest-mattermost-organizer-guide.md]:
```suggestion
- Join the **Off-Topic** channel to have conversations with all Hacktoberfest event organizers and attendees.
```

**Review by justinegeffen — DISMISSED** [review]

**Comment by justinegeffen** [comment]
Just some minor suggestions. Otherwise it's good to go. :)

**Comment by jasonblais** [comment]
Thank you!

**Pull request merged**